### ISSUE TITLE
Fix documentation for dlstest.DlsClient

### ISSUE THREAD

**Issue by daniel-zullo** [sociomantic-tsunami/dlsproto#33]
The documentation in the examples/tests for the method `connectionNotifier()` in module `dlstest.DlsClient` is incorrect for the tag `neo-alpha-1`

```D
        /***********************************************************************

            Connection notifier used by the client (see the outer class' ctor).

            Params:
                node_address = address/port of node which notification refers to
                e = exception instance indicating an error (null indicates
                    connection success)

        ***********************************************************************/

        private void connectionNotifier ( RawClient.Neo.ConnNotification info)
        {
            with (info.Active) switch (info.active)
            {
            case connected:
                log.trace("Neo connection established (on {}:{})",
                    info.connected.node_addr.address_bytes,
                    info.connected.node_addr.port);

                if (this.connect_task && this.connect_task.suspended())
                    this.connect_task.resume();

                break;
            case error_while_connecting:
                with (info.error_while_connecting)
                {
                    this.connection_error = true;
                    log.error("Neo connection error: {} (on {}:{})",
                            getMsg(e),
                            node_addr.address_bytes, node_addr.port);
                }
                break;
            default:
                assert(false);
            }
        }
```

**Comment by nemanja-boric-sociomantic** [comment]
Btw, there's also stuff in `src/dlsproto/client/UsageExamples.d` which you may find useful.

**Comment by daniel-zullo** [comment]
Addressed in #34
